payments/models.py

```diff
@@ -512,8 +533,12 @@ def save(self, *args, **kwargs):
         # If the product instance is being passed
         # Price has to be assigned before saving but only if creating
         if creating and self.product:
-            # Assign the price and tax from the product
-            price = self.product.price_value
+            # Assign the tax from the product
+            price = (
+                self.product.price_value
+                if self.product and hasattr(self.product, "price_value")
+                else 0
```

Comment:
Would it be safer to fallback to `None` in this case? Maybe the configuration error would be more visible then and something would hopefully blow up in payment process? 😄

Reply:
Then we would need to assign it to `price_value` somewhere else, for winter storage orders this is the place where it's set.